const errHandler = {
  error(app, logger) {
    app.use(async (ctx, next) => {
      try {
        await next();
      } catch (err) {
        logger.error(err);
        ctx.status = 500;
        ctx.body = "ðŸ˜­å“‡å“¦ç½‘ç«™å‡ºé”™äº†~";
      }
    })
  }
}

module.exports = errHandler;